### PR TITLE
ZF new releases and plane

### PR DESCRIPTION
**Pull request by gianarb**

### CODE REVIEW

File: site/2016/03/23/zf-doctrine-module-release-and-future.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+It's new securty release:
+
+- Drop compatibility with PHP 5.4 `#553 <https://github.com/doctrine/DoctrineModule/pull/553>`_
+- Update doctrine/common ~2.6 `#551 <https://github.com/doctrine/DoctrineModule/pull/551>`_
+
+DoctrineModule 1.0.1
+---------------------
+
+- Drop compatibility with PHP 5.4 `#553 <https://github.com/doctrine/DoctrineModule/pull/553>`_
+- Improve TravisCi build to work with PHP 7 `#553 <https://github.com/doctrine/DoctrineModule/pull/553>`_
```
Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: Both have the same url ?

File: site/2016/03/23/zf-doctrine-module-release-and-future.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
+Zend Framework and doctrine, releases and future
+================================================
+
+We are working hard to support all change that Zend Framework done and are
+doing, the community are working very hard to increase the value of each single
+components and we are doing our best to follow this trend.
+
+We decided to do only one post with 4 similar releases, 2 for DoctrineModule, 1
```
Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: to only do ?

File: site/2016/03/23/zf-doctrine-module-release-and-future.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
+Zend Framework and doctrine, releases and future
+================================================
+
+We are working hard to support all change that Zend Framework done and are
```
Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: has done ?

File: site/2016/03/23/zf-doctrine-module-release-and-future.rst [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- `#146 <https://github.com/doctrine/DoctrineMongoODMModule/pull/146>`_ Support for `odm:schema:update` command
+- `#153 <https://github.com/doctrine/DoctrineMongoODMModule/pull/153>`_ Update to stable package versions
+- `#133 <https://github.com/doctrine/DoctrineMongoODMModule/pull/133>`_ Allows null value as DefaultDb
+- `#155 <https://github.com/doctrine/DoctrineMongoODMModule/pull/155>`_ Fix pagination count with eager cursors
+- `#156 <https://github.com/doctrine/DoctrineMongoODMModule/pull/156>`_ Remove support for old PHP versions
+- `#160 <https://github.com/doctrine/DoctrineMongoODMModule/pull/160>`_ Fixed Zend\Mvc 2.7 compatibility
+
+About the future
+----------------
+We start to work on DoctrineModule 2.0, this release will provide support for
```
Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: We will start ?

File: site/2016/03/23/zf-doctrine-module-release-and-future.rst [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+PHP ~5.6 and PHP 7.0, and it will support Zend Framework ~3.0.
```
Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: and will support ?

File: site/2016/03/23/zf-doctrine-module-release-and-future.rst [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+*DoctrineModule* 0.x will be update only with security fix, we suggest you to
```
Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: will only be updated with security fix, we suggest you update

File: site/2016/03/23/zf-doctrine-module-release-and-future.rst [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+update becuase the codebase it's practically the same and we will stop to
```
Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: *because

Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: the codebase is pratically the same. The security support will end on the 29/01/2019.

File: site/2016/03/23/zf-doctrine-module-release-and-future.rst [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+support it from 29/Jannuary/2019.  We are working on DoctrineModule 1.x to
+support ZF 2.6 and 3.0 components, this is one of the main goal of this major
+release.
+
+About DoctrineORMModule and DoctrineMongoODMModule we have in plane for both a
```
Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: we plan a first stable release for both. They will support ZF 2.6 and 3.0 and the release will happen when DoctrineModule will support ZF2.6 and 3.0 too.

File: site/2016/03/23/zf-doctrine-module-release-and-future.rst (hunk 5 above)
Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: will only support ?

File: site/2016/03/23/zf-doctrine-module-release-and-future.rst [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+first stable release when the supports for ZF 2.6 and 3.0 when DoctrineModule
+will be release with ZF 2.6 and 3.
+After that we will start to work on 2.0.0 in order to support DoctrineModule 2.0
```
Comment (mikeSimonson) [inline site/2016/03/23/zf-doctrine-module-release-and-future.rst]: I am not sure to understand that sentence.
Maybe its a bit too far in the future?

**Comment by mikeSimonson** [comment]
@gianarb I made a few comment on text. I think that it makes it more readable but keep in mind that I am not a native speaker.

Cheers

**Comment by alcaeus** [comment]
Closing, this seems hardly relevant.

**Pull request closed without merging**